scripts/data_analysis/constants/common.py

```diff
@@ -25,3 +35,47 @@ def extract_data(url=URL):
     else:
         print(url, " responded with ", response.status_code)
         return '{}'
+
+def findParentSchema(response_dict, schema_name):
+    """
+    Search all schemas for any reference to target schema (i.e. find all direct parents) and return results in a dataframe.
```

Comment:
Wonder if `schema_name` should be named `target_schema` to follow the language of the function description? Or maybe in the description just add `schema_name` to link the two?

Reply:
I think both should be changed to child_schema actually, to make it more descriptive